interfaces/fetch.idl

```diff
@@ -5,16 +5,18 @@
 
 typedef (sequence<sequence<ByteString>> or record<ByteString, ByteString>) HeadersInit;
 
-[Constructor(optional HeadersInit init),
- Exposed=(Window,Worker)]
+[Exposed=(Window,Worker)]
 interface Headers {
+  constructor(optional HeadersInit init);
```

Comment:
@saschanaz FYI looks like `= {}` wasn't added here. (I presume this comes from your fixup script.)

Reply:
@foolip HeadersInit is not a dictionary here so it's not needed 👍